sechub-cli/src/daimler.com/sechub/cli/constants_filepatterns_gen.go

```diff
@@ -16,23 +16,4 @@ to git repository even when its generated  */
 // DefaultZipAllowedFilePatterns - Defines file patterns to include in zip file.
 // These patterns are considered as source code to be scanned.*/
 var DefaultZipAllowedFilePatterns = []string{
-    ".apex", ".apexp", ".component", ".object", ".page", ".report", ".tgr", ".trigger", ".workflow", /* Apex */
```

Comment:
This file is important for the SecHub client. Was it intentional to remove the lines in this file?

Reply:
Sorry, this slipped through by mistake. Reversed it. Done.

Reply:
@sven-dmlr : Maybe we should remove the ..._gen.go parts from git, commit and then add to .gitignore. https://github.com/Daimler/sechub/issues/522 describes this.